### ISSUE TITLE
Hope to support import asset from in Window/Package Manager/Import 

### ISSUE THREAD

**Issue by mrkyee** [Code-Stage/Package2Folder#6]
I'm sorry for forgetting to reply for so long. Currently, it seems that only a local .unitypackage file can be supported by dragging it into the Import Asset window. However, I usually import resources from Window/Package Manager/Import. In this case, I cannot specify the import path. Is there any plan to support this situation?

**Comment by DmitriyYukhanov** [comment]
Hi,

Thanks for getting back with a reply.

Do you mean **My Assets** tab in the Package Manager?

I believe all usual (real) packages are installed into the `PackagesCache` folder (or sit in the `Packages` folder) and changing their import path seems to have no sense.

**Comment by mrkyee** [comment]
Yes.But i need to find PackagesCache folder .I want to do everything in unity rather than searching for them in folders

**Comment by DmitriyYukhanov** [comment]
I am not sure if I fully understand, sorry.

`/Library/PackageCache` folder is not intended to be altered and all real packages in the project which do install in there are no intended to be placed under the `/Assets` folder.

Most Asset Store packages sitting in _My Assets_ tab of the Package Manager are made as vanilla `.unitypackage` and install into the `/Assets` folder.

However, as it's now allowed to publish assets as embedded packages to the Asset Store, some packages from the _My Assets_ tab are actually real packages with special inner structure, and they will install into the project's `/Packages` folder (they become embedded packages when installed).

From what I can see, Package2Folder can be helpful for those assets which are not embedded packages, but regular assets placed inside `/Assets` folder.

Package2Folder could be helpful only in case when you need to change the installation path for the old "vanilla" assets which extract to `/Assets` folder.

Does it sound like something you're asking about @mrkyee ?

**Comment by mrkyee** [comment]
I'm not sure how to explain my question, but here's an example. I recently purchased a resource from the Unity Asset Store, and it will be displayed in the Package Manager under "My Assets".
If I want to import it into my project, obviously I need to take the following steps:
 1. Download it locally. 
![image](https://user-images.githubusercontent.com/35887438/234600203-b430ad36-f228-40b3-ae9c-d96d335cbaf9.png)

 2. Click "Import". 
 
![image](https://user-images.githubusercontent.com/35887438/234600359-71a21ba4-8cf1-4acf-8174-7f7bb3f1962e.png)

 3. At this point, a window will appear showing the resources that need to be imported. However, it's often at this step that I run into problems, because the publisher may not have put the package in the standard plugin directory, and that's where my frustration lies.

So my request is to support this situation, because often when I import resources, it's through this way, rather than going to the cache directory to find the corresponding .unitypackage file and drag it into the Assets directory.

**Comment by DmitriyYukhanov** [comment]
Got it now, thanks for explaining it in all details.

Yeah, that would be nice to have indeed, but currently I don't have any ideas on how to inject into the importing process to route import into a different folder.

This needs some investigation to find out if it's possible. Unfortunately, I'm too tightly packed now to give any ETAs for this, so please just follow this issue for any updates.

**Comment by mrkyee** [comment]
You're too kind. Looking forward to your next update.

**Comment by rwetzold** [comment]
You might want to take a look at my tool which mostly replaces the package manager and uses package2folder under the hood to allow importing anything into custom directories. And best, not only the whole package but any part of it including all dependencies.

https://assetstore.unity.com/packages/tools/utilities/asset-inventory-2-275893
